src/components/Shared/VulnerabilityPackageCard.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Card, Stack, Typography } from '@mui/material';
+import makeStyles from '@mui/styles/makeStyles';
+
+const useStyles = makeStyles(() => ({
+  cvePackageCard: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    background: '#FFFFFF',
+    boxShadow: '0rem 0.3125rem 0.625rem rgba(131, 131, 131, 0.08)',
+    border: '1px solid #E0E5EB',
+    borderRadius: '0.75rem',
+    flex: 'none',
+    alignSelf: 'stretch',
+    width: '100%'
+  },
+  cveInfo: {
+    marginTop: '2%'
+  }
+}));
+
+function VulnerabilityPackageCard(props) {
+  const { cve } = props;
+  let styles = useStyles();
```

Comment:
const, plus let's keep the naming consistent between components, so please rename the variable as "classes"

Reply:
Thanks! Addressed!